src/MockClassifier.Api/Controllers/InstitutionController.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using MockClassifier.Api.Models;
+
+namespace MockClassifier.Api.Controllers
+{
+    [Route("input-from-dmr/institution")]
```

Comment:
Apologies, I missed this in the docs but I think this should be `/classify` (drop `input-from-dmr`). Currently posting to the `input-from-dmr/institution` resource implies (in REST terms) that you're the creating/posting a new institution.